pkg/util/util.go

```diff
@@ -99,3 +100,11 @@ func (u *Util) GetSystemNSDomains() []string {
 	}
 	return domains
 }
+
+// Shared Functions
+func HomeDir() string {
```

Comment:
Minor: Can we add a function comment starting with the function name similar to other comments in this file?